dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/support/MockInvoker.java

```diff
@@ -194,9 +194,11 @@ public static Object getMockObject(ExtensionDirector extensionDirector, String m
                 + ", please check if there's mock class or instance implementing interface "
                 + serviceType.getName(), e);
         }
-        if (mockClass == null || !serviceType.isAssignableFrom(mockClass)) {
-            throw new IllegalStateException("The mock class " + mockClass.getName() +
-                " not implement interface " + serviceType.getName());
+        if (mockClass == null) {
+            throw new IllegalStateException("not implement interface " + serviceType.getName());
```

Comment:
log out `mockService` to help users find out why

Reply:
mockService perhaps null

Reply:
How about log out if is not null?